src/site/content/en/blog/community-focus-bramus/index.md

```diff
@@ -0,0 +1,90 @@
+[Scroll-Linked Animations](https://css-tricks.com/scroll-linked-animations-with-the-web-animations-api-waapi-and-scrolltimeline/), 
+and [Container Relative Lengths](https://www.bram.us/2021/09/21/css-container-queries-container-relative-lengths/).
+{% endAside %}
+
+**Rachel:** What was your route into web development?
+
+**Bramus:** As a kid, I always liked to tinker with things. I would spend days playing with my LEGO® bricks, building my own fantasy world and objects from scratch.
+
+When we got a computer at home—an unusual device to own in the 1990s—I soon traded in the physical toys with computer games. I wasn't an avid gamer though; I don't think I ever finished a game entirely. Instead of finishing the games, I found myself modding them. 
+
+In 1997, while looking up information about those games and tools, I also discovered `view-source`. Curious to know how things were built, I started collecting HTML-snippets of the sites that I visited. Combining those snippets with Frontpage Express (an application that came with Internet Explorer 4 and 5), I soon created my very first web pages with info about myself. Those pages never got published, they only existed on one of the floppy disks I carried around.
+
+From that time on I continued to become more interested in computers and the web. This interest led me to flunk a year in high school on purpose, so that I could switch major from economics to IT—I knew I wanted to pursue a career in IT. By 2002 I was in college, where I properly learned HTML and took my first steps into CSS and JavaScript. During those three years I realized that the web was my true passion, and in 2005, fresh out of college, I took on my first job as a professional web developer.
+
+## On being a front and backend developer
+
+**Rachel:** I spotted on your site that you are both a front and backend developer, I followed a similar path being originally a Perl, then a PHP and MySQL developer. Do you feel more excited by one side or the other? Do you think the possibility of being a hybrid developer is vanishing given the complexity of learning just one part of the stack?
+
+**Bramus:** Throughout my career I've constantly been floating between backend and frontend. One year I would find myself elbow-deep into JavaScript and React (and even React Native), only to be creating Terraform scripts and Docker containers the year after. I like mixing the two, yet my passion always lay with the frontend, and CSS specifically.
+
+In the early days of tinkering with the web, one simply was the "webmaster" and did it all. As the scope of the work was pretty limited back then, it was quite easy to keep up. Having seen both frontend and backend explode over the past 20 years, it became harder and harder to maintain expertise across the field. That's why I decided to mainly focus on frontend again in 2020.
```

Comment:
What would you think about including an editor's footnote on the term "webmaster"? That was the term used in early web, and really until recently was still the primary term. GDDSG states we [should not use webmaster](https://developers.google.com/style/word-list#webmaster) and lists alternatives.

I propose we add a footnote at the bottom which states the following:

> In this interview, Bramus used the term "webmaster," which was the common title used in the early days of the web. We've committed to using inclusive language, and per the [Google Developer Documentation Style Guide](https://developers.google.com/style/word-list#webmaster) we recommend the name "website administrator" or "website owner" instead.

Reply:
I'm not convinced we need to call it out, it's why I kept it in quotes. I think if someone was referring to it as their modern day job title it might be a bit different, but it's clearly a historical usage.

Reply:
Fair enough. I don't feel so strongly about it, and the quotes are probably sufficient enough without us editorializing.